### PR TITLE
RHDEVDOCS-5720: Updating the uninstall section

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve this issue: https://issues.redhat.com/browse/RHDEVDOCS-5720

**Aligned team**: DevTools

**Version for cherrypicking**: build-docs-1.0

**Content for preview**: https://68189--docspreview.netlify.app/openshift-builds/latest/uninstalling/uninstalling-openshift-builds#ob-deleting-the-builds-component-and-custom-resources_uninstalling-openshift-builds

**SME review**:  @apoorvajagtap @jkhelil 

**QE review**:  @jitendar-singh 

**Peer review**: @agantony

### CODE REVIEW

File: modules/ob-deleting-the-builds-component-and-custom-resources.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ Delete the custom resources (CRs) that were created by default during Operator i
 .Procedure
 . In the *Administrator* perspective of the web console, navigate to *Administration* -> *Custom Resource Definition*.
 
-. Type `operator.shipwright.io` in the *Filter by name* box to search for the {builds-operator} CR.
+. Type `shipwright.io` in the *Filter by name* box to search for the {builds-operator} CR.
```
Comment (agantony) [inline modules/ob-deleting-the-builds-component-and-custom-resources.adoc:18]:
```suggestion
. In the *Filter by name* box, enter `shipwright.io` to search for the {builds-operator} CR.
```

**Review by agantony — COMMENTED** [review]
Good job! Added a small suggestion; overall lgtm!

**Comment by apoorvajagtap** [comment]
LGTM.

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**